src/consts/consts.py

```diff
@@ -114,7 +114,7 @@ def all(cls):
 DEFAULT_LOAD_BALANCER_PORT = 6443
 DEFAULT_IPXE_SERVER_PORT = 8500
 DEFAULT_TANG_SERVER_PORT = 7500
-DEFAULT_IPXE_SERVER_IP = "192.168.122.1"
+DEFAULT_IPXE_SERVER_IP = "10.0.132.14"
```

Comment:
for afterwards: checking if the existing ipxe flow has been effected by it

Reply:
it should take the private IP of the machine instead, but it's hardcoded at the moment :(